bullet-train.zsh-theme

```diff
@@ -352,10 +355,11 @@ prompt_git() {
     fi
     prompt_segment $BULLETTRAIN_GIT_BG $BULLETTRAIN_GIT_FG
 
+    eval git_prompt=${BULLETTRAIN_GIT_PROMPT_CMD}
     if [[ $BULLETTRAIN_GIT_EXTENDED == true ]]; then
-      echo -n $(git_prompt_info)$(git_prompt_status)
+      echo -n ${git_prompt}$(git_prompt_status)
```

Comment:
You can replace `${git_prompt}` with `${BULLETTRAIN_GIT_PROMPT_CMD}`, making `local git_prompt` no longer necessary.